data/ecosystem/adopters.yaml

```
@@ -20,6 +20,11 @@
   components: [Collector, Python]
   reference: ''
   contact: ''
+- name: Delivery Hero
+  url: https://www.deliveryhero.com/
+  components: [Collector, Go, Java]
+  reference: ''
+  contact: 'https://github.com/ekovalenk0'
```

Btw, here and elsewhere, we don't need quotes around URLs.
```suggestion
  contact: https://github.com/ekovalenk0
```